mlfinlab/tests/test_sampling.py

```diff
@@ -86,21 +86,25 @@ def test_seq_bootstrap(self):
         """
 
         non_nan_meta_labels = self.meta_labeled_events.dropna()
-        ind_mat = get_ind_matrix(non_nan_meta_labels)
+        ind_mat = get_ind_matrix(non_nan_meta_labels, self.data)
 
         label_endtime = non_nan_meta_labels.t1
-        bar_index = list(non_nan_meta_labels.index)  # generate index for indicator matrix from t1 and index
+        trimmed_price_bars_index = self.data[(self.data.index >= non_nan_meta_labels.index.min()) &
+                                             (self.data.index <= non_nan_meta_labels.t1.max())].index
+        bar_index = list(non_nan_meta_labels.index)  # Generate index for indicator matrix from t1 and index
         bar_index.extend(non_nan_meta_labels.t1)
-        bar_index = sorted(list(set(bar_index)))  # drop duplicates and sort
+        bar_index.extend(trimmed_price_bars_index)
+        bar_index = sorted(list(set(bar_index)))  # Drop duplicates and sort
         ind_mat_book_implementation = book_ind_mat_implementation(bar_index, label_endtime)
```

Comment:
To this function: `book_ind_mat_implementation`

Please add in the docstring that it is snippet 4.3 pg 65.